docs/metrics/doid.md

```diff
@@ -32,11 +32,11 @@
 
 | Metric | Value |
 | ------ | ----- |
-| AnnotationAssertion | 99692 |
-| SubAnnotationPropertyOf | 1 |
+| AnnotationAssertion | 103824 |
```

Comment:
> However, in "components/doid.owl" synonyms that are abbreviations, e.g. AIDS for DOID:635 are still missing.

Uhoh. That's not what I observed yesterday. Let me double check and see if I can figure out what's going on there.

Reply:
@twhetzel I am seeing that AIDS is there. I would guess that you are not seeing it because the `components/` are `.gitignore`'ed, and if you did refresh the component, maybe you weren't on this branch, or maybe something else went wrong? I don't think `-B` is necessary for refreshing components goals, but you could try that.

<details><summary>Screenshot</summary>
<p>

<img width="1031" alt="Screenshot 2024-09-21 at 5 32 50 PM" src="https://github.com/user-attachments/assets/4673bfa4-6349-45b7-a29b-69bf259faca0">

</p>
</details>

Reply:
I updated the original comment (re-posted below) so all is good.
| Update: The component files are in the .gitignore so not expected in this diff. I'll run the "components/doid.owl" locally. Confirmed the abbreviations are now present.